src/components/InputTodo.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/prop-types */
+import React, { useState } from 'react';
+import { FaPlusCircle } from 'react-icons/fa';
+
+const InputTodo = (props) => {
```

Comment:
- It would be better to implement prop de-structuring here as it makes the code DRYer and easy to read. 
   > i.e
    `const InputTodo = ({ addTodoProps }) => {`
- Hence you should also add prop validations for the `inputTodo()` func as this ensures that the value passed   is of correct data type.
    > i.e
   ...
   `};`
   `InputTodo.propTypes = {`
   `addTodoProps: Proptypes.func.isRequired,`
  ` };`
   `export default InputTodo;`